AzurePipelinesToGitHubActionsConverter/AzurePipelinesToGitHubActionsConverter.Core/Conversion/PipelineProcessing.cs

```diff
@@ -245,7 +245,7 @@ public GitHubActionsRoot ProcessPipeline(AzurePipelinesRoot<T, T2> azurePipeline
             else if (azurePipeline.parameters != null)
             {
                 //For now, convert the parameters to variables
-                gitHubActions.env = ProcessSimpleVariables(azurePipeline.parameters);
+                gitHubActions.env = ProcessSimpleVariables(azurePipeline.parameters.ToDictionary(k => k.name, v => v.@default));
```

Comment:
Parameter object in azure pipelines can be more complicated than a keyvaluepair<string,string>, here I am trying to retain backwards compatibility to get your lib to build at this stage, it will probably require another generic but I'm not finished yet.